build/scripts/build.ps1

```diff
@@ -133,7 +138,7 @@ function Run-MSBuild([string]$projectFilePath, [string]$buildArgs = "", [string]
         $args += " /m"
     }
 
-    if ($noAnalyzers -or ($cibuild -and $testVsi)) {
+    if ($skipAnalyzers) {
         # Avoid spending time in analyzers when requested, and also in the slowest integration test builds
```

Comment:
I suspect the "slowest integration test builds" part of the comment should move with your code?

(I'd also suspect we might not care about that...?)